.circleci/scripts/linux_test_benchmark.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
Can you please create an issue to track the removal of redundant scripts.

Reply:
I will create an issue and all relevant details as soon as this PR is merged